src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -975,25 +975,6 @@ public class ShowHideConsoleMenuItemConverter : IValueConverter
         #endregion
     }
 
-    public class ShowHidePreviewBubblesConverter : IValueConverter
```

Comment:
@QilongTang ... this is a public class and public methods, I'm not sure these can be removed without introducing BC breaks?